tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+              </conditional>
+              <param name="obs_collection" type="text" label="Collection" help="Name of the observation collection" />
+              <param name="obs_title" type="text" label="Observation title (archive specific)" />
+              <param name="obs_id" type="text" label="Observation Id" />
+              <param name="facility_name" type="text" label="Facility name" help="Name of the facility used for this observation" />
+              <param name="instrument_name" type="text" label="Instrument name" help="Name of the instrument used for this observation" />
+              <param name="em_min" type="text" label="Start of the energy range, vacuum wavelength in meters" />
+              <param name="em_max" type="text" label="Stop of the energy range, vacuum wavelength in meters" />
+              <param name="obs_publisher_id" type="text" label="Publisher dataset ID" />
+              <param name="s_fov" type="text" label="Diameter (bounds) of the covered region (deg)" />
+              <param name="calibration_level" type="select" label="Calibration level (0, 1, 2, 3, 4, 5)" >
+                <option value="none">None</option>
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+              </param>
+              <param name="order_by" type="select" label="Order by" help="Field to use for ordering the results">
+                <option value="none" selected="true">No specific order</option>
+                <option value="size" >Order by size (Ko)</option>
+                <option value="collection" >Order by collection</option>
+                <option value="object" >Order by Target Name, alphabetically</option>
+              </param>
+            </when>
+            <when value="raw_query">
+              <param name="table" type="text" label="Table name" help="Name of the table you want to query (FROM clause)" />
+              <section name="where_clause" title="WHERE clause (field = value)" expanded="false">
+                <param name="where_field" type="text" label="WHERE field" help="WHERE **field** = value" />
+                <param name="where_condition" type="text" label="WHERE value" help="WHERE field = **value**" />
+              </section>
+              <param name="url_field" type="text" label="Url field" help="Table field containing the url of the file to download" />
+            </when>
+          </conditional>
+        </section>
+        <param name="number_of_files" type="integer" value="1" label="Number of files or urls to download" help="Min = 1 Max = 100; Beware of disk space usage when downloading large number of files!" />
+        <param name="csv_file" type="boolean" truevalue="true" falsevalue="false" label="Download URLs as CSV"/>
```

Comment:
this is an additional download, additional to FITS, correct? maybe mention that?

Reply:
It's one of the options, just like downloading FITS.
```suggestion
        <param name="csv_file" type="boolean" truevalue="true" falsevalue="false" label="Return URL list as CSV"/>
```

Reply:
Ok, it that case just make sure an output is actually selected. Not that by accident none of the outputs is selected. An alternative modeling of that UI would be a select field with optional=false (the default) and maybe as `display` checkboxes https://docs.galaxyproject.org/en/latest/dev/schema.html#id46

Reply:
Sounds good, this would be useful. @francoismg can you add it?